live_pipeline/pipeline_functions.py

```diff
@@ -0,0 +1,324 @@
+from datetime import datetime as dt
+import pytz
+from os import environ
+import json
+from boto3 import client
+from dotenv import load_dotenv
+from psycopg2 import connect
+from psycopg2.extras import RealDictCursor
+import requests
```

Comment:
`requests` is not installed with `pip`. I think it should go before `boto3` and after `json`